script.js

```diff
@@ -0,0 +1,45 @@
+const hamburger = document.getElementById('menuimg');
+const menu = document.getElementById('fullmenu');
+const names = document.getElementById('logonav');
+const closingbutton = document.getElementById('closingbutton');
+const envelope = document.getElementById('envelope');
+const body = window.innerWidth;
+const item1 = document.getElementById('menuitemport');
+const item2 = document.getElementById('menuitemabout');
+const item3 = document.getElementById('menuitemcontact');
```

Comment:
- [ OPTIONAL ] Great job on working on the reqested changes by the previous code reviewer but as I can see here when I press on portolfio or about or contact inside the menu it doesn't close it and I can't see the section that took me to it unless I press on close button so kindly Fix that so when I press on these link it closes and take me to the section so you can see the importance of the menu in action ✅✅💪🏻💪🏻🔥🔥

Reply:
> * [ OPTIONAL ] Great job on working on the reqested changes by the previous code reviewer but as I can see here when I press on portolfio or about or contact inside the menu it doesn't close it and I can't see the section that took me to it unless I press on close button so kindly Fix that so when I press on these link it closes and take me to the section so you can see the importance of the menu in action ✅✅💪🏻💪🏻🔥🔥

I think is a little error, i have a if for blocking that the mobile menu work if the user isn't in the mobile version, if you open it in desktop size and resized instead of using the inspect tool the size of the client don't update in the variable that keep track of that and blocks the event listener of those elements.